openedx_webhooks/tasks/example.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""
+Example Celery task, can be used for testing the Celery infrastructure
+at Heroku.
+
+Example usage::
+
+    from openedx_webhooks.tasks.example import add
+    add.delay(1, 2)
+"""
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
```

Comment:
If we have to put boilerplate `__future__` imports that we don't need at the top of the file, can we at least put it on one line instead of three?